src/components/homepage/Footer.js

```diff
@@ -0,0 +1,33 @@
+import React from "react";
+import "./Footer.css";
+const Footer = () => {
+  return (
+    <div>
+      <footer className="footer">
+        <h3 className="footer__copy">
+          &#169; 2022 Copyright all right reserved
+        </h3>
+        <h4>Made with ❤ Anjali Chauhan</h4>
+        <ul className="f-connectme">
+          <li>
```

Comment:
Here, as these links are getting repeated. You can map over an array, like this
```javascript
[{
    _id: uuid(), 
    link: "https://github.com/anjali1102", 
    linkFor: "github"
    }, {...others}].map(({_id, link, linkFor}) => <li>
<a href={link}><i className={`fa fa-${linkFor}`}></i></a>
</li>
```

In this way, code will not repeat.